shell/platform/windows/window_top_level_message_handler.cc

```diff
@@ -0,0 +1,74 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "window_top_level_message_handler.h"
+
+#include <TlHelp32.h>
+#include <Windows.h>
+#include <WinUser.h>
+#include <tchar.h>
+
+#include "flutter/shell/platform/windows/flutter_windows_engine.h"
+
+namespace flutter {
+
+static constexpr char kExitTypeCancelable[] = "cancelable";
```

Comment:
Could we move this string out of this class? Ideally all strings for Flutter framework messages are contained in the `PlatformHandler`. Perhaps introduce an enum or boolean?